src/library_pthread.js

```diff
@@ -400,6 +400,28 @@ var LibraryPThread = {
         'workerID': PThread.nextWorkerID++,
 #endif
       });
+    }),
+
+    loadWasmModuleToAllWorkers: function(onMaybeReady) {
+#if !PTHREAD_POOL_SIZE
+      onMaybeReady();
+#else
+      // Instantiation is synchronous in pthreads and we assert on run dependencies.
+      if (
+        ENVIRONMENT_IS_PTHREAD
+#if WASM_WORKERS
+        || ENVIRONMENT_IS_WASM_WORKER
+#endif
+      ) return onMaybeReady();
```

Comment:
```suggestion
      ) {
        return onMaybeReady();
      }
```
(Also existing in the code before the copy+changes, but let's clean it up.)